torch/csrc/inductor/aoti_torch/oss_proxy_executor.cpp

```diff
@@ -140,7 +452,10 @@ OSSProxyExecutor::OSSProxyExecutor(const std::string& json_path, bool is_cpu) {
   json_file >> json_obj;
 
   // Access data
+  std::cout << json_obj["nodes"] << std::endl;
```

Comment:
Are these ```cout```s for debugging or logging? If they are for logging, could we use c10's logging utilities?